packages/api/src/schema/schema.yaml

```diff
@@ -1662,58 +1677,43 @@ components:
           description:
             How many seconds the duration of each output segment should be
         catalystPipelineStrategy:
-          $ref: "#/components/schemas/task/properties/params/properties/upload/properties/catalystPipelineStrategy"
-
+          $ref: >-
+            #/components/schemas/task/properties/params/properties/upload/properties/catalystPipelineStrategy
     task:
       type: object
-      table: task
       additionalProperties: false
       properties:
         id:
           type: string
-          readOnly: true
-          index: true
           description: Task ID
           example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
         userId:
           type: string
-          readOnly: true
-          index: true
           description: User ID of the task owner
           example: 66E2161C-7670-4D05-B71D-DA2D6979556F
         type:
           type: string
           description: Type of the task
           enum:
             - upload
-            - import # Import is the deprecated pipeline, upload is on Catalyst
+            - import
             - export
-            - transcode # Transcode is the deprecated pipeline, used for deprecated `/api/asset/transcode` and `/api/asset/:assetId/transcode` endpoints
```

Comment:
Are those comments invalid? Saw others been removed as well, I thought it was valid yaml syntax.